platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,19 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, partial=True):
```

Comment:
DRF defaults to `partial=False`. I think we ought to do the same. Partial updates must be PATCH.

https://github.com/encode/django-rest-framework/blob/master/rest_framework/mixins.py#L66